analyses/independent-samples/run-independent-samples.sh

```diff
@@ -24,7 +24,7 @@ else
 fi
 
 independent_DNA_each="results/independent-specimens.wgswxspanel.primary-plus.eachcohort.tsv"
-independent_DNA_all="results/independent-specimens.wgswxspanel.primary-plus.allcohorts.tsv"
+independent_DNA_all="results/independent-specimens.wgswxspanel.primary-plus.tsv"
```

Comment:
I think this is an unexpected change

Reply:
I think this was an error from before. Sorry about that! At one point I added `allcohorts` to all the independent samples but that was later removed and when running with `--all_cohorts`, the file is left as what it is (without additional information about cohorts). Hence this file `results/independent-specimens.wgswxspanel.primary-plus.allcohorts.tsv` no longer exist in our current workflow and the code would not run with it either.